packages/terra-application-navigation/src/terra-dev-site/test/application-navigation/HeroAndUser.test.jsx

```diff
@@ -0,0 +1,24 @@
+/* eslint-disable */
+import React from 'react';
+import ApplicationBase from 'terra-application/lib/application-base';
+import ApplicationNavigation from '../../../ApplicationNavigation';
+import Placeholder from 'terra-doc-template/lib/Placeholder';
+
+const hero = <Placeholder title="Test Hero" variant="light" />;
+
+const userConfig = {
+  name: 'Test Name',
+  detail: 'Test Detail',
+};
+
+// TODO: remove terra-application after it is incorporated into dev-site
```

Comment:
I this this TODO has been resolved: https://github.com/cerner/terra-dev-site/blob/master/package.json#L110

Reply:
Updated.